msteams-platform/resources/teams-updates.md

```diff
@@ -31,33 +31,36 @@ The Classic Teams client is represented using the `teams` field and the new Team
 
 ## Timelines and rollout
 
-To ensure a smooth transition, a phased rollout of the new platform is planned as follows:
+To ensure a smooth transition, a phased rollout of the new platform is planned.
 
-* **Availability of all platform features from Classic Teams**: All apps are available in the new Teams Client. All platform features except the features listed under [known issues](#known-issues) are now available. Platform features under known issues aren't currently supported in the new Teams client.
+**Availability of all platform features from Classic Teams**: All apps are available in the new Teams Client. All platform features except the features listed under [limitations](#limitations) are now available.
 
-## Known issues
+## Limitations
 
 > [!NOTE]
 >
 > * The new Teams client is supported in Windows and Mac clients and Government Community Cloud (GCC) and GCC-High environments.
-> * We recommend you to test the functionality of  apps, tabs, messaging extensions, bots, and link unfurling on Windows and Mac clients and GCC and GCC-High environments after switching from the Classic Teams client to the new Teams client.
-> * [Adaptive Card tabs](../tabs/how-to/build-adaptive-card-tabs.md) aren't supported in the new Teams client. The Classic Teams client is expected to be deprecated by March 31, 2024. If your app is using Adaptive Card tabs, we recommend you rebuild the tab as a web-based tab. For more information, see [build tabs for Teams](../tabs/what-are-tabs.md).
+> * We recommend you to test the functionality of apps, tabs, messaging extensions, bots, and link unfurling on Windows and Mac clients and GCC and GCC-High environments after switching from the Classic Teams client to the new Teams client.
 
 The following Teams features aren't currently supported in the new Teams client:
 
-* [Location](../concepts/device-capabilities/location-capability.md#location-apis) and [Media](../concepts/device-capabilities/media-capabilities.md#media-capability-apis) APIs aren't supported in the new Teams client. We recommend using HTML5 Geolocation, and Media.
-
 * [DevTools](/microsoft-edge/devtools-guide-chromium/overview) isn't supported in the new Teams client.
-  
+
 * [App caching](~/tabs/how-to/app-caching.md) isn't supported in the new Teams client.
 
 * [Static tabs for group chat, channels, and meetings](~/tabs/how-to/create-personal-tab.md#extend-static-tabs-to-group-chat-channels-and-meetings) aren't supported in the new Teams client.
```

Comment:
We can delete this line now. Static tabs for group chats, channels, and meetings are now supported in the New Teams Client (in preview but that matches the Classic Teams Client).

Reply:
@ydogandjiev,

For the content related to static tabs, I have created another PR. This would be addressed through PR #10655.